docs/ExpectAPI.md

```diff
@@ -239,8 +239,12 @@ It's also possible to create custom matchers for inline snapshots, the snapshots
 const {toMatchInlineSnapshot} = require('jest-snapshot');
 
 expect.extend({
-  toMatchTrimmedInlineSnapshot(received) {
-    return toMatchInlineSnapshot.call(this, received.substring(0, 10));
+  toMatchTrimmedInlineSnapshot(received, inlineSnapshot) {
+    return toMatchInlineSnapshot.call(
+      this,
+      received.substring(0, 10),
+      inlineSnapshot,
+    );
```

Comment:
```suggestion
  toMatchTrimmedInlineSnapshot(received, ...rest) {
    return toMatchInlineSnapshot.call(
      this,
      received.substring(0, 10),
      ...rest,
    );
```

not sure if that's better or not... I don't think we'll be adding any more args, but who knows

Reply:
Yeah my initial proposal would throw on the initial run (`toMatchTrimmedInlineSnapshot()`) since `toMatchInlineSnapshot` relies on arity. While your patch fixes it, it does so by completely bypassing arity because `arguments.length` is 0 for rest spread.

Reply:
sounds like some work could be done to make custom snapshot matchers easier to use... Can land this for now tho as it's more correct

Reply:
I'd like to improve these since I find them pretty powerful. E.g. https://github.com/facebook/jest/issues/10546, https://github.com/facebook/jest/issues/10888 or custom, async inline snapshot matchers (currently throwing).

Reply:
PRs welcome 😀 I'm unable to keep on top of the incoming issues, it's very random if I look at them or not... PRs are much easier to get through (even though that also has a backlog currently 🙈)